build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
         }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.2.1'
+        classpath 'com.android.tools.build:gradle:3.3.3'
```

Comment:
can we update to 4.x directly since that's the latest ? It might mean some more changes but should not be a huge lift.